### ISSUE TITLE
Camera, MOAICamera support

### ISSUE THREAD

**Issue by MattiaFortunati** [ymobe/rapanui#55]
RapaNui needs support to Moai's camera.

We need to check MOAICamera and how to implement it in RapaNui.

**Comment by BlueByLiquid** [comment]
I would like to voice my support for this features It will make things so much nicer

**Comment by niom** [comment]
Modified AngryDogsAgainstMoais in rapanuidev branch to use basic MOAICamera2D. Has basic camera moving and following the target.

**Comment by MattiaFortunati** [comment]
Yes, thanks to niom, now we have a sample in which is showed how to use MOAICamera2d ;D
However it would be nice to have a dedicated sample,too. ^^ (I'm writing here as a reminder)

**Comment by niom** [comment]
You are right. I am also planning to create separate camera tutorials which we can use to create proper RNCamera layer.

**Comment by vertti** [comment]
What would be awesome for the camera would be:
- Follow a prop (camera keeps the prop in the center of the view)
- Bounded camera (for example camera follows a prop but never so far that it shows over a background image)
